cmd/cape/cmd/run.go

```diff
@@ -78,19 +85,33 @@ func run(cmd *cobra.Command, args []string) error {
 		return UserError{Msg: "error retrieving file flag", Err: err}
 	}
 
+	checksumArg, err := cmd.Flags().GetString("checksum")
+	if err != nil {
+		return UserError{Msg: "error retrieving checksum flag", Err: err}
+	}
+
+	// Deprecated
 	funcHashArg, err := cmd.Flags().GetString("function-hash")
```

Comment:
do we need this still if we're using checksum instead now?

Reply:
This is so if any of our friendlies are still using it it will work but they'll see the message to swap over